rocAL/rocAL/include/augmentations/effects_augmentations/node_jitter.h

```diff
@@ -25,18 +25,17 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-
-class JitterNode : public Node
-{
+class JitterNode : public Node {
 public:
     JitterNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     JitterNode() = delete;
-    void init(int kernel_size);
-    void init(IntParam *kernel_size);
+    void init(int kernel_size, int seed);
```

Comment:
why is the seed only there for this node?

Reply:
RPP Jitter and Noise augmentations accepts seed as an input
Hence we have added them in rocAL as well